account-kit/react/src/components/button.tsx

```diff
@@ -17,31 +16,31 @@ type ButtonProps = (
     "variant" | "ref"
   >;
 
-export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
```

Comment:
forward ref was unnecessary here, so removing it to try and optimize a bit of memory. had no impact really